mk/config.mk

```diff
@@ -176,6 +176,31 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary.
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA_PATHS to the paths to one or more TA ELF
```

Comment:
@etienne-lms ultimately I think CFG_ is not very convenient. During testing I have changed several times the list of applications (adding one or two, changing them), and with CFG_ the tee.bin is fully recompiled each time and it's a bit painful.
So I'm not changing the prefix. We can change later depending on the use cases.